docs/source/deployment/google_cloud_run.rst

```diff
@@ -67,7 +67,7 @@ This guide uses the IrisClassifier BentoService from the :doc:`Quick start guide
 
 .. code-block:: bash
```

Comment:
Add a little bit more context to `This guide uses the IrisClassifier BentoService from the :doc:`Quick start guide <../quickstart>`:` ?